tests/test_read_30.py

```diff
@@ -15,16 +15,22 @@
 logger.setLevel(logging.DEBUG)
 
 
-@pytest.mark.skip(reason="lasio.exception.LASHeaderError in ~Test_Data section")
 def test_read_v30_sample():
     las = lasio.read(stegfn("3.0", "sample_3.0.las"))
     assert las.version[0].mnemonic == "VERS"
     assert las.version[0].value == 3.0
     assert las.version[0].descr == "CWLS LOG ASCII STANDARD -VERSION 3.0"
 
 
-def test_read_v30_sample_without_test_sections():
-    las = lasio.read(stegfn("3.0", "sample_3.0_without_test_sections.las"))
-    assert las.version[0].mnemonic == "VERS"
-    assert las.version[0].value == 3.0
-    assert las.version[0].descr == "CWLS LOG ASCII STANDARD -VERSION 3.0"
+
+
+def test_read_v30_sample_v20_sections_are_empty():
+    """
+    Verify 'Curves' doesn't read 'Core_*' sections
+    Verify 'Parameter' doesn't read 'Performations_*' sections
+    """
+    las = lasio.read(stegfn("3.0", "sample_3.0.las"))
+    assert las.sections['Curves'] == []
+    assert las.sections['Parameter'] == []
```

Comment:
Maybe this is personal preference but I get worried by ``== []`` - can we change it to something explicit like ``assert not las.sections['Curves']`` or perhaps ``len(las.sections['Curves']) == 0``? Not sure.

Reply:
Good point!  I think `assert not las.sections['Curves']` would be good and work, but I'll check it out and make a change based on the findings.